automation/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/azure/AzureRuntimeSpec.scala

```diff
@@ -86,16 +90,25 @@ class AzureRuntimeSpec
         )(implicitly, GeneratedLeonardoClient.runtimeInStateOrError(ClusterStatus.RUNNING))
 
         _ <- loggerIO.info(
-          s"AzureRuntime: runtime ${workspaceId}/${runtimeName.asString} create monitor result: $monitorCreateResult"
+          s"AzureRuntimeSpec: runtime ${workspaceId}/${runtimeName.asString} create monitor result: $monitorCreateResult"
         )
         _ = monitorCreateResult.getStatus() shouldBe ClusterStatus.RUNNING
 
         _ <- loggerIO.info(
-          s"AzureRuntime: runtime ${workspaceId}/${runtimeName.asString} delete starting"
+          s"AzureRuntimeSpec: runtime ${workspaceId}/${runtimeName.asString} delete starting"
         )
         // Delete the runtime
         _ <- IO(runtimeClient.deleteAzureRuntime(workspaceId, runtimeName.asString, true))
 
+        _ <- loggerIO.info(
+          s"AzureRuntimeSpec: about to test proxyUrl"
+        )
+//TODO
```

Comment:
what's this TODO for?

Reply:
oops, was to call the proxy during development, removed